src/generators/static/packageFileGenerator.ts

```diff
@@ -103,20 +138,39 @@ function restLevelPackage(packageDetails: PackageDetails) {
       })
     },
     devDependencies: {
-      autorest: "latest",
       "@microsoft/api-extractor": "^7.18.11",
-      "@types/node": "^14.14.22",
+      autorest: "latest",
+      "@types/node": "^12.0.0",
```

Comment:
are you sure the version is still valid ?

Reply:
looks like we are downgrade several packages versions.

Reply:
I think the exisiting packages use this version, such as https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/purview/purview-catalog-rest/package.json#L101